gitlab/_backends/requests_backend.py

```diff
@@ -117,3 +122,26 @@ def http_request(
             **kwargs,
         )
         return RequestsResponse(response=response)
+
+    def response_content(
```

Comment:
The prefix "response_" is redundant

Reply:
Keep in mind this method is in `RequestsBackend`, not in `RequestsResponse` as the GitHub diff suggests. Did you still mean this prefix is redundant in the backend class?

Reply:
Thanks for the clarification. I totally missed it.  In that case, what is the connection between the `response_content` function and the class `RequestsBackend`?

Reply:
The implementation is backend-specific, so it can't live in utils anymore if we want to add other backends. For example requests has `iter_content`, whereas httpx has `iter_bytes` and `iter_text`.

Putting it into the backend class was an easy minimal copy/paste, eventually it would probably make sense to remove this method and handle it in the response itself I assume as I say above:

> Needed in order to later add other backends that handle streaming responses differently. Just a first step, later maybe we could get rid of response_content completely from individual methods and just handle this in the request code :thinking:  

Hope that makes sense:) I can push a branch with an httpx backend to show the different implemenations if needed.

Reply:
Thanks again for the clarification. I am totally fine with going forward with this approach as long as we agree that it might be refactored in the future.
Only thing I would add the staticmethod decorator which will eliminate the need of instantiating the class.

Reply:
I agree with that, my thinking is as I said,`eventually it would probably make sense to remove this method and handle it in the response itself`, but maybe there are other ways :)

Ah good idea, will add that, completely missed that while copy pasting.

Reply:
Done, thanks @lmilbaum!